tests/system/src/test/java/test/robot/javafx/scene/DatePickerOnFocusLostTest.java

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright (c) 2018, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
it's a brand new file, I wonder if it should have 2023 only

Reply:
Unless this file was substantially copied from another test, it should be 2023 only.